evm/src/interfaces/INttManager.sol

```diff
@@ -43,12 +49,41 @@ interface INttManager is INttManagerState {
     /// @param mode The mode.
     error InvalidMode(uint8 mode);
 
+    /// @notice Error when the refund to the sender fails.
+    /// @dev Selector 0x2ca23714.
+    /// @param refundAmount The refund amount.
     error RefundFailed(uint256 refundAmount);
-    error TransceiverAlreadyAttestedToMessage(bytes32 nttManagerMessageHash);
+
+    /// @notice Error when the tranceiver already attested to the message.
+    ///         To ensure the client does not continue to initiate calls to the ateestationReceived function.
+    /// @dev Selector 0x2113894.
+    /// @param nttManagerMessageHash The hash of the message.
+    error TrasceiverAlreadyAttestedToMessage(bytes32 nttManagerMessageHash);
+
+    /// @notice Error when the message is not approved.
+    /// @dev Selector 0x451c4fb0.
+    /// @param msgHash The hash of the message.
     error MessageNotApproved(bytes32 msgHash);
+
+    /// @notice Error if the target chain is not supported or does not exist.
```

Comment:
This isn't true actually. The error is emitted when trying to execute a message on an unintended chain: https://github.com/wormhole-foundation/example-native-token-transfers/blob/73c42314aeb1ba58df7f74f9134485c5778ffe2f/evm/src/NttManager/NttManager.sol#L137-L140